projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following-recommendations.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * REST API endpoint to retrieve following recommendations
+ *
+ * @package automattic/jetpack
+ * @since $$next-version$$
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Following_Recommendations
+ */
+class WPCOM_REST_API_V2_Endpoint_Following_Recommendations extends WP_REST_Controller {
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'following';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->wpcom_is_wpcom_only_endpoint    = true;
+		$this->wpcom_is_site_specific_endpoint = false;
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/recommendations',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_response' ),
+					'permission_callback' => 'is_user_logged_in',
```

Comment:
Since this will be used in the block editor, maybe we can restrict access to the endpoint even more, only to folks who have contributor or author capabilities?